tests/core/CoreTest.py

```diff
@@ -514,7 +517,7 @@ def test_run_result_handler_exception(self):
         on_result = unittest.mock.Mock(side_effect=ValueError)
 
         with self.assertLogs(logging.getLogger()) as cm:
-            run({bear}, on_result)
+            self.execute_run({bear}, on_result)
```

Comment:
Nah, I think its better to remove it. Doesn't make sense to put an unnecessary overhead.